test/notebooks/notebook_test_utils.py

```diff
@@ -113,6 +116,29 @@ def init_notebook_test():
     nb_client.execute()
 
 
+
+# ~/git/ai-lab/test/notebooks/notebook_test_utils.py
+def set_log_level_for_libraries(level=logging.WARNING):
+    modules = cleandoc(
+        """
+        traitlets
+        luigi-interface
```

Comment:
it is actually weird that you see this at all, because the itde handles its logging itself and should by default log only on warning

Reply:
However, it could be also, because it runs in pytest where logging is again a bit weird.